src/App.css

```diff
@@ -0,0 +1,16 @@
+@import url("https://grow-n-know.netlify.app/styles/main.css");
```

Comment:
import any external css file in the html for better performance.

Reply:
will do that